framework/encode/capture_manager.h

```diff
@@ -49,6 +49,11 @@
 GFXRECON_BEGIN_NAMESPACE(gfxrecon)
 GFXRECON_BEGIN_NAMESPACE(encode)
 
+class ApiCaptureManagerBase;
+class VulkanCaptureManager;
```

Comment:
Do you need the API-specific classes referenced here?  I don't see them used below.